modules/machineset-azure-accelerated-networking.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+
+[id="machineset-azure-accelerated-networking_{context}"]
+= Accelerated Networking for Microsoft Azure VMs
+
+Accelerated Networking uses single root I/O virtualization (SR-IOV) to provide Microsoft Azure VMs with a more direct path to the switch. This results in enhanced network performance. This feature can be enabled during or after installation.
+
+[id="machineset-azure-accelerated-networking-limits_{context}"]
+== Limitations
+
+Consider the following limitations when deciding whether to use Accelerated Networking:
+
+* Accelerated Networking is supported on clusters with installer-provisioned infrastructure only.
```

Comment:
Is this implicitly because we don't have Machine API on UPI clusters? I think as a product we are trying to stray away from differing between IPI and UPI and so maybe we want to phrase this in a way that suggests only available on clusters using machinesets?